testinfra/vars/mon-staging.yml

```diff
@@ -98,4 +98,4 @@ log_events_with_ossec_alerts:
     alert: >
       ossec: Agent started: 'app->10.20.2.2'.
     level: "1"
```

Comment:
Hmm, this is producing a test failure.. Looks like it's 3 (it is for me on my prod VM).